network-rfc/11_tokenomics_v2.1.md

```diff
@@ -0,0 +1,78 @@
+# Tokenomics 2.1
+
+## Overview
+
+Tokenomics 2.1 addresses the centralization issues of Tokenomics 2.0, namely:
+
+- That there is a single (de-facto centralized) pool for providing SQD for yield, which makes SQD a security
+- The lack of dynamic pricing and many parameters that has to be hard-coded/adjusted in a centralized (or, at best, DAO-like) fashion. The subscription fee is not established by an open market and the marketplace. 
+- Removing the treasury-initiated buyback-and-burn mechanics which makes SQD a security
+- Moving the reward token out of scope
+- Introducing the fee switch to the Portals (to be activated in the future if necessary)
+- Making it possible to register Portals on EVMs (in particular, Base) and Solana. For Solana users, it opens up the posibility to pay in USDC or SOL.
+
+
+## The SQD Flows
+
+**Workers** 
+Workers serve the data and receive rewards in SQD. The reward depend on the number of served queries, the amount of delegated tokens and the uptime of the worker. The worker has to lock 100k SQD to participate in the network. The maximal amount of rewards distributed per worker and the delegators is controlled by a single parameter called `TARGET_APR`. The reward is then split abetween the worker and the delegators for the worker.  
+
+**Delegators**
+Delegators delegate SQD to workers to get a part of the worker reward. Both the amount of delegated tokens and the served queries affect the reward per query served. 
+
+**Data consumers**
+Data consumers query the network p2p using a Portal. The maximal bandwidth that can be consumed by a Portal is determined by the amount of SQD locked in the Portal contract. Thus, the data consumer either buys SQD on the open market and locks the desired amount themselves, or makes an _SQD Provision Offer_ to SQD holders willing to provide SQD in return to the fee. 
+
+An SQD Provision Offer is an agreement to lock SQD for a specified amount of time for a fee, paid as continuosly during the whole lock period. The fee is locked by the consumer in advance and can be paid in any of the supported tokens. Special conditions apply for extending the provision offer and withdrwals. 
+
+**SQD providers**
+SQD providers hold SQD and fullfill the matching _SQD Provision Offers_. Active providers can advertise their target fees in advance to make the market and set the expectations for the data consumers. 
+
+## Emission reduction
+
+The `TARGET_APR` currently set to 25% yearly, will be gradually reduced and replaced by the fees collected from the portals. 
```

Comment:
In what form will the fees be rewarded to the workers? If everyone pays fees in different currencies, how (most importantly, at what price) do we convert them?